frame/bags-list/src/mock.rs

```diff
@@ -91,7 +91,7 @@ frame_support::construct_runtime!(
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		System: frame_system::{Pallet, Call, Storage, Event<T>, Config},
-		BagsList: bags_list::{Pallet, Call, Storage, Event<T>},
+		BagsList: bags_list::{Pallet, Call, Storage, Event<T, I>},
```

Comment:
Here I would add two copies of this bags list pallet, and then do a test which modifies the pallet in different ways and shows they both hold state correctly.

Generally, any PR should have tests which show what the PR does.

Reply:
we can have a proper and realistic test for this after #10944, in the meantime I can't think of any imaginative way tbh. I think as long as this PR does not alter any logic, it is fine to go in without further tests. We don't want to test "does pallet instances way the work we expect" here.